sidebars.json

```diff
@@ -517,11 +492,12 @@
         "id": "reference-landing"
       },
       "items": [
+        "pulsar-api-overview",
+        "reference-rest-api-overview",
```

Comment:
Since `pulsar-api-overview` is in 1st level, then`reference-rest-api-overview` should be in 2nd level, no?